### PR TITLE
Add "mc admin policy info" command to display policy content

### PR DESCRIPTION
**Pull request by donatello**

### CODE REVIEW

**Comment by donatello** [comment]
Additionally this PR removes the same functionality which was available as `mc admin policy list TARGET POLICYNAME` to make the command line easier to use.

**Comment by donatello** [comment]
Could I get some review on this? Need this for upcoming changes to minio and mc policy, users and groups.

File: cmd/admin-policy-info.go [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+}
+
+// checkAdminPolicyInfoSyntax - validate all the passed arguments
+func checkAdminPolicyInfoSyntax(ctx *cli.Context) {
+	if len(ctx.Args()) != 2 {
+		cli.ShowCommandHelpAndExit(ctx, "info", 1) // last argument is exit code
+	}
+}
+
+// mainAdminPolicyInfo is the handler for "mc admin policy list" command.
```
Comment (krisis) [inline cmd/admin-policy-info.go]: nit:
```suggestion
// mainAdminPolicyInfo is the handler for "mc admin policy info" command.
```

File: cmd/admin-policy-info.go [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+	// Create a new MinIO Admin Client
+	client, err := newAdminClient(aliasedURL)
+	fatalIf(err, "Cannot get a configured admin connection.")
+
+	policies, e := client.ListCannedPolicies()
+	fatalIf(probe.NewError(e).Trace(args...), "Cannot list policy")
+
+	if len(policies[policyName]) != 0 {
+		printMsg(userPolicyMessage{
+			op:         "list",
```
Comment (krisis) [inline cmd/admin-policy-info.go]: should it be,
```suggestion
			op:         "info",
```

Comment (donatello) [reply cmd/admin-policy-info.go]: Yeah, and we need to implement the printing logic. Its done now @krisis

**Comment by krisis** [comment]
@donatello https://github.com/minio/mc/blob/master/docs/minio-admin-complete-guide.md#command-policy---manage-canned-policies needs to be updated.

**Comment by donatello** [comment]
Updated docs

**Pull request merged**